### ISSUE TITLE
Expectations

### ISSUE THREAD

**Issue by danpowell88** [mdasberg/ng-apimock#49]
It would be quite handy to be able to verify the mock was called with certain parameters or certain endpoints were hit, I can't currrently see a way to provide this.

**Comment by wswebcreation** [comment]
Hi @danpowell88 

Isn't this more like a unit test of the source code instead of testing this in ng-apimock?

**Comment by danpowell88** [comment]
Yeah I'd think mostly unit testing would cover a lot of it but it is handy in some circumstances to be able to check that the api was called.

nock does it but it doesn't have any kind of nice angular support from what i could tell

https://github.com/node-nock/nock

**Comment by wswebcreation** [comment]
Tnx for your answer. ng-apimock doesn't have a prebuild assertion in it, but it could be a feature. @mdasberg , what do you think about that?

**Comment by lucasbento** [comment]
This would be a really good feature, also looking forward for this one.
